qatrack/qa/testpack.py

```
@@ -66,7 +67,7 @@ def create_testpack(test_lists=None, cycles=None, extra_tests=None, description=
         },
         'meta': {
             'version': settings.VERSION,
-            'datetime': "%s" % (timezone.now().astimezone(timezone.utc)),
+            'datetime': "%s" % (timezone.now().astimezone(datetime.UTC)),
```

Ruff (0.15.18)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)